petric.py

```diff
@@ -248,6 +248,7 @@ def get_image(fname):
 
 DATA_SLICES = {
     'Siemens_mMR_NEMA_IQ': {'transverse_slice': 72, 'coronal_slice': 109},                 # 'sagittal_slice': 89
```

Comment:
Should we add it here too?


```suggestion
    'Siemens_mMR_NEMA_IQ': {'transverse_slice': 72, 'coronal_slice': 109, 'sagittal_slice': 89}
```

Reply:
Could do, but the pngs on the website didn't have it. Maybe less confusing to keep it as is.